.github/workflows/ci.yml

```diff
@@ -34,10 +34,10 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - uses: manusa/actions-setup-openshift@v1.1.2
+      - uses: redhat-actions/oc-installer@v1
```

Comment:
The reason why I choose this action is that it setup network specifically to the GitHub Actions infra so it's likely not supported by the Red Hat action 👎

Reply:
yes, figured that out when workflow fails. closing :(